test/install_test.go

```diff
@@ -268,30 +264,15 @@ func TestInject(t *testing.T) {
 
 	prefixedNs := TestHelper.GetTestNamespace("smoke-test")
 
-	if TestHelper.AutoInject() {
-		out, err = testutil.ReadFile("testdata/smoke_test.yaml")
-		if err != nil {
-			t.Fatalf("failed to read smoke test file: %s", err)
-		}
-		err = TestHelper.CreateNamespaceIfNotExists(prefixedNs, map[string]string{
```

Comment:
not necessarily in this PR, but consider testing manual injection as well. if we're going to continue supporting it, we should exercise cli injection somewhere in our integration tests.

Reply:
Agreed. I added #2744 to track that.